src/scripts/surf.ts

```diff
@@ -0,0 +1,74 @@
+// Description:
+//  Surf server stat comparison
+//
+// Commands:
+//  hubot surf me - compare surfers
+//
+// Author:
+//  Steve Shipsey
+
+import Axios, { AxiosResponse } from 'axios';
+import * as Cheerio from 'cheerio';
+import { Response, Robot } from 'hubot';
+import { table, TableUserConfig } from 'table';
+
+const baseUrl = 'https://snksrv.com/surfstats';
+const surfers = [
+  'STEAM_1:0:6030042',
+  'STEAM_1:0:1166568',
+];
+const headers = ['Surfer', 'Points', 'Map Completions'];
+
+const fetchSurfer = async (steamId: string) => await Axios.get<any>(`${baseUrl}`, {
+  params: {
+    id: steamId,
+    view: 'profile',
+  },
+});
+
+const parse = (cheerio: any): string[] => {
+  const $ = Cheerio.load(cheerio.data);
+  return [
+    $('h2').first().text(),
+    getTextWithoutChildren($('td').first()).slice(2),
+    getTextWithoutChildren($('td').eq(2)).slice(2),
+  ];
+};
+
+const sortByPoints = (surfDataColumns: string[][]) =>
+  surfDataColumns.sort((a, b) => parseInt(b[1], 10) - parseInt(a[1], 10));
```

Comment:
technically, I think this sort by code will parseInt the same string more than once. Doesn't matter at all here. Just something to note.